examples/example-gateway/config/production.json

```diff
@@ -20,6 +20,9 @@
 	"clients.bar.port": 4001,
 	"clients.bar.ip": "127.0.0.1",
 
+	"clients.SimpleService.port": 4002,
```

Comment:
We need to use something from client-config.json, like the "name" field ( https://github.com/uber/zanzibar/blob/master/examples/example-gateway/clients/googlenow/client-config.json#L2 ).

The thrift file is not the source of truth, the client-config is.